CONTRIBUTING.md

```diff
@@ -19,11 +19,26 @@ Redmine project. Look for tickets with [easy] or [easy-hack] tags.
 
 ## How to get this repository working
 
-Upon setting up a new openQA instance, it's also necessary to install some aditional dependencies that are inherent to this repository,
-for which there are two ways:
+Upon setting up a new openQA instance, it's also necessary to install some
+aditional dependencies that are inherent to this repository.
+
+* On openSUSE to install an openQA worker and all dependencies do:
+
+```
+zypper in os-autoinst-distri-opensuse-deps-worker perl-JSON-Validator gnu_parallel
+```
+
+* If you just want to just install the dependencies without the openQA worker:
+
+
+```
+zypper in os-autoinst-distri-opensuse-deps perl-JSON-Validator gnu_parallel
```

Comment:
Hmmm I guess JSON-Validator and gnu_parallel should be also part of the `os-autoinst-distri-opensuse-deps`, but I'm ok to merge as that's a separate thing